src/04/ToggleButton.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
What do you say about having this file be empty? Maybe it's better to only have the scaffolding in the first file and remove them in 01, 02, and 03 as well?